src/settings/settingsdialog.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>800</width>
-    <height>552</height>
+    <width>837</width>
+    <height>586</height>
```

Comment:
Were these changes intentional?

Reply:
I may have changed it by accident and will correct it